src/test-app/__tests__/testSagaWithState.test.ts

```diff
@@ -10,49 +10,45 @@ import { getCount } from '../selectors';
 
 nock.disableNetConnect();
 
-test('Test helper with mocked sleep call', async () => {
-  return testSagaWithState(
-    watchForUserSelectorToCountIfNotChangedWithing3s,
-    userSelected({ id: 2 }),
-    [calls(sleep).receiving()],
-    undefined,
-    userReducer,
-    { count: 1, selectedUser: 2, usersById: {} },
-  );
+test('Test helper with mocked sleep call', () => {
+  return expectSaga(watchForUserSelectorToCountIfNotChangedWithing3s)
```

Comment:
@HenriBeck These all don't have the `sleep` call mocked and hence take 3s each:

```
  ✓ Test helper with mocked sleep call (3014ms)
  ✓ Test helper with mocked select (3005ms)
  ✓ Test helper asserting on message (3006ms)
  ✓ Test helper asserting on message, fails of message doesn't match (3009ms)
```

I'll try to add that, the facilities should be there.

Reply:
We could add a warning when a test takes longer than 1 second. The default timeout is currently set to 10 seconds by default; we should at least decrease it to 5 seconds in my opinion.

Reply:
I'll change it down to 1 for these test and then we should just mock the `sleep`. Otherwise they just take too long to run all the time. OK?

Reply:
(With that change we also know that the mocking works.)

Reply:
Yeah, but I was talking about also changing the default timeout

Reply:
Mocking works like a charm to speed these up 👍 

```
 PASS  src/test-app/__tests__/expectSaga.test.ts
  ✓ Test helper with mocked sleep call (10ms)
  ✓ Test helper with mocked select (2ms)
  ✓ Test helper asserting on message (1ms)
  ✓ Test helper asserting on message, fails of message doesn't match (6ms)
```